Packs/CommonTypes/Layouts/layoutscontainer-Account.json

```diff
@@ -1,376 +1,779 @@
 {
-	"id": "accountRep",
-	"group": "indicator",
-	"name": "Account Indicator",
-	"description": "",
-	"version": -1,
-	"fromVersion": "6.0.0",
+    "id": "accountRep",
+    "group": "indicator",
+    "name": "Account layout by XSOAR",
```

Comment:
why is this the name?

Reply:
I was asked to change all names to 6.0 layouts